.github/workflows/nightly.yml

```diff
@@ -26,82 +28,35 @@ jobs:
             build_type: Debug
             static: true
             container: '{ "image": "rippleci/clio_ci:latest" }'
-    uses: ./.github/workflows/build_impl.yml
-    with:
-      runs_on: ${{ matrix.os }}
-      container: ${{ matrix.container }}
-      conan_profile: gcc
-      build_type: ${{ matrix.build_type }}
-      code_coverage: false
-      static: ${{ matrix.static }}
-      unit_tests: true
-      integration_tests: true
-      clio_server: true
-      disable_cache: true
-
-  run_tests:
-    needs: build
-    strategy:
-      fail-fast: false
-      matrix:
-        include:
-          - os: macos15
-            conan_profile: apple_clang_16
-            build_type: Release
-            integration_tests: false
-          - os: heavy
-            conan_profile: gcc
-            build_type: Release
-            container:
-              image: rippleci/clio_ci:latest
-            integration_tests: true
-          - os: heavy
-            conan_profile: gcc
-            build_type: Debug
-            container:
-              image: rippleci/clio_ci:latest
-            integration_tests: true
-    runs-on: [self-hosted, "${{ matrix.os }}"]
-    container: ${{ matrix.container }}
-
-    services:
-      scylladb:
```

Comment:
We don't need to have scylladb for unit tests, that's why in test_impl the service is only run for integration tests

Reply:
This was already only running for integration. If integration_tests was not set to true the image would resolve to empty string and the whole service would be discarded. It's a hack so i like your approach better anyway.

Reply:
I think the idea was that we only call it with either integration_tests or unit_tests for any given run. I may be misremembering though 👍